homeassistant/components/climate/ephember.py

```diff
@@ -96,9 +105,16 @@ def target_temperature_step(self):
     @property
     def current_operation(self):
         """Return current operation ie. heat, cool, idle."""
-        if self._zone['isCurrentlyActive']:
-            return STATE_HEAT
-        return STATE_IDLE
+        return STATE_AUTO
+
+    @property
+    def operation_list(self):
+        """List of available operation modes."""
+        return [STATE_AUTO]
```

Comment:
A list of only one possible operation mode is not very useful - what is the user supported to set if there is only one choice? Also, none of these choices is returned by `current_operation_mode`…

Reply:
In order for this device to be viewable from google assistant it must support operation mode. Currently the device only supports being set to one mode which is AUTO, so that is what is returned here. 

I don't see any `current_operation_mode` function in the climate devices. There is `current_operation` which returns `STATE_AUTO`

Reply:
Sorry, I meant `current_operation`. I think the real fix for this problem would be to make google assistant also see devices which don't support setting an operation mode, instead of introducing "dummy operation modes" into all climate platforms which currently don't support this.

If we were to do that, the UI would show every user for every climate device (that doesn't actually support setting operation modes) an "operation mode" selection field which is of no use, which is probably annoying and confusing.

Reply:
From the discussion I had with @balloob in #13158 the google assistant trait requires being able to report a mode, so unless that changes this seems like the best workaround. 

Note: I have found that the underlying api for this device does support other operation modes. I will look to add that support over the next couple of weeks to flesh out this implementation. For now I would just like to get the assistant support back.